packages/webview_flutter/tizen/src/webview.h

```diff
@@ -27,7 +27,7 @@ class WebView : public PlatformView {
  public:
   WebView(flutter::PluginRegistrar* registrar, int viewId,
           flutter::TextureRegistrar* textureRegistrar, double width,
-          double height, flutter::EncodableMap& params);
+          double height, flutter::EncodableMap& params, void* winHandle);
```

Comment:
```suggestion
          double height, flutter::EncodableMap& params, void* win_handle);
```

Reply:
Changed same as the name of the platform view.